package.json

```diff
@@ -25,26 +30,16 @@
     "tslib": "^1.10.0"
   },
   "devDependencies": {
+    "remark-cli": "^7.0.1",
+    "remark-preset-lint-recommended": "^3.0.3",
     "prettier": "^1.19.1",
     "typescript": "^3.7.5",
-    "rollup": "^1.18.0",
```

Comment:
Doesn’t the lock file need to be updated after this?

Reply:
Thought it did, but I'll check again. But most importantly, I forgot to remove the rollup config file!